src/ffi/orx.ml

```diff
@@ -935,10 +940,10 @@ module Clock = struct
 
   let register
       ?(handle = Clock_callback_store.default_handle)
+      ?(module_id = Module_id.Main)
```

Comment:
These are the defaults mentioned above. Other values can still be used, but this should reduce noise in most call sites.